Add Community Organisers Activities page (#1332)
EuroPython/website: src/content/pages/community-activities.mdx

This year, we are dedicating a two-hour slot in the Open Spaces to all community organisers who want to share their experiences. More
importantly, it's a chance to explore how the EuroPython Society can support you in achieving your
goals and addressing your current challenges.

* Room: TBD
* Time: TBD

## Community Organisers Lunch

Do you know all the organizers from other European Python communities and conferences?
If not, this is a great opportunity to meet more people who, just like you, are doing their best to keep Python events and communities alive—so that more people have the chance to learn, connect, and grow with Python!

We'll be meeting on Friday, July 18, at 12:55 in the Zoom room on the first floor near the registration area for a special lunch—an opportunity to connect with others who share your passion.

## Community Booths

Each year, we reserve booth space for communities, initiatives, nonprofit organizations, and open source projects that support the Python community and the wider open source ecosystem.
Join us in the Exhibit Hall during the main conference days to meet this year’s amazing Community Booth lineup!

> <Button url="#" class="disabled">Sign Up For a Booth!</Button>

Author: the `disabled` part I think doesn't work, and before we merge this, we need to add the url for submitting requests for booths

This year, we are dedicating a two-hour slot in the Open Spaces to all community organisers who want to share their experiences. More
importantly, it's a chance to explore how the EuroPython Society can support you in achieving your
goals and addressing your current challenges.

* Room: TBD
* Time: TBD

![](./images/pyo-open-space.jpg)

## Community Organisers Lunch

Do you know all the organizers from other European Python communities and conferences?
If not, this is a great opportunity to meet more people who, just like you, are doing their best to keep Python events and communities alive—so that more people have the chance to learn, connect, and grow with Python!

We'll be meeting on Friday, July 18, at 12:55 in the Zoom room on the first floor near the registration area for a special lunch—an opportunity to connect with others who share your passion.

## Community Booths

Each year, we reserve booth space for communities, initiatives, nonprofit organizations, and open source projects that support the Python community and the wider open source ecosystem.
Join us in the Exhibit Hall during the main conference days to meet this year’s amazing Community Booth lineup!

> <Button url="#" class="disabled">Sign Up For a Booth!</Button>

Reviewer: ```suggestion
<Button url="https://forms.gle/p2cG6YAXxoq1qRd69">Sign Up For a Booth!</Button>
```

---
title: Community Organisers Activities
url: /community-activities
subtitle:

>   Booths, Open spaces, and more activities for organisers

Reviewer 2: ```suggestion
  Booths, open spaces, and more activities for organisers
```

---

# Community Organisers Activities

> The Python community is an essential part of the language, and for many people
> the reason for sticking around and keep meetups, conferences, forums, and many
> more things running, in order to help others.
> 
> This year at EuroPython, we have a few activities focused on some of the
> communities around Europe and world, but also initiatives around Python.

Reviewer: ```suggestion
The Python community is an essential part of the language, and for many people, it's
the reason they stick around and keep meetups, conferences, forums, and so much more running to help others.

This year at EuroPython, we have several activities focused on communities across Europe and around the world, as well as initiatives centered around Python itself. 
```

## Community Organisers Open Space

Communication is the key to avoid repetition and failing on the same mistakes.
Let's be together and share the experience of our communities and our personal

> challenges, and wins! so more communities can improve.

Reviewer 2: ```suggestion
challenges and wins so more communities can improve.
```

> This year, we are saving a 2-hour slot in the Open Spaces for all the community

Reviewer 2: ```suggestion
This year, we are saving a two-hour slot in the Open Spaces for all the community
```

> Communication is the key to avoid repetition and failing on the same mistakes.
> Let's be together and share the experience of our communities and our personal
> challenges and wins so more communities can improve.
> 
> This year, we are saving a two-hour slot in the Open Spaces for all the community
> and conference organisers that want to share their experiences, and more
> importantly, to see how the EuroPython Society can help you to achieve your
> goals and solve your current issues.

Reviewer: ```suggestion
Communication is key to avoiding repetition and learning from past mistakes.
Let's come together and share the experiences of our communities, as well as and our personal
challenges and successes, so that more communities can grow and improve.

This year, we are dedicating a two-hour slot in the Open Spaces to all community organisers who want to share their experiences. More
importantly, it's a chance to explore how the EuroPython Society can support you in achieving your
goals and addressing your current challenges.
```

* Room: TBD
* Time: TBD

## Community Organisers Lunch

> Do you know all the organisers from other European communities and conferences?
> If not, you have a good opportunity to meet more people, that like you, are
> doing their best for keep more Python events and communities running,
> so more people have the chance of learning and improving at Python!

Reviewer: ```suggestion
Do you know all the organizers from other European Python communities and conferences?
If not, this is a great opportunity to meet more people who, just like you, are doing their best to keep Python events and communities alive—so that more people have the chance to learn, connect, and grow with Python!
```

> We will be meeting at TBD during TBD for a special lunch, where you will be

Reviewer 3: Friday, 18 July, 12:55, Zoom, First floor (entrance near the registration area :))

> We will be meeting at TBD during TBD for a special lunch, where you will be
> able to relax and get to know more people with the same passion as yours.

Reviewer: ```suggestion
We'll be meeting on Friday, July 18, at 12:55 in the Zoom room on the first floor near the registration area for a special lunch—an opportunity to connect with others who share your passion.
```

## Community Booths

Each year, we set aside booth space for communities, initiatives, nonprofit

> organizations, and community open source projects that serve the Python

Reviewer 2: ```suggestion
organisations, and community open source projects that serve the Python
```

> Each year, we set aside booth space for communities, initiatives, nonprofit
> organisations, and community open source projects that serve the Python
> community and the broader open source ecosystem. Join us in the Exhibit Hall
> during the main talk days of the conference to welcome this year's wonderful
> Community Booth lineup!

Reviewer: ```suggestion
Each year, we reserve booth space for communities, initiatives, nonprofit organizations, and open source projects that support the Python community and the wider open source ecosystem.
Join us in the Exhibit Hall during the main conference days to meet this year’s amazing Community Booth lineup!
```

<Button url="#" class="disabled">Sign Up For a Booth!</Button>

## PyLadies Open Space & Lunch

> Check [the PyLadies activities page](/pyladies) for more information.

Reviewer: ```suggestion
Check out [the PyLadies activities page](/pyladies) for more information.
```
